.schema.env

```diff
@@ -1,6 +1,7 @@
 
 STREAM_CHUNK_SIZE=1000
 CLINICAL_PROGRAM_UPDATE_TOPIC=PROGRAM_UPDATE
+RDPC_PROGRAM_UPDATE_TOPIC=song_analysis
```

Comment:
let's include the `RDPC_URL` here too.

also make sure to update dev helm values before merging to develop ;)